packages/network-controller/tests/helpers.ts

```diff
@@ -87,7 +89,7 @@ export function buildNetworkControllerMessenger(
 ): NetworkControllerMessenger {
   return messenger.getRestricted({
     name: 'NetworkController',
-    allowedActions: [],
+    allowedActions: ['ErrorReportingService:captureException'],
```

Comment:
Here we need to add `ErrorReportingService:captureException` to the list of external actions because we are now using it in the controller.